pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kubecmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	latestapi "github.com/openshift/origin/pkg/api/latest"
+	client "github.com/openshift/origin/pkg/cmd/client"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Rolls back a deploymentConfig to match a previous deployment.
+
+By default, only the pod template of the previous deployment is carried over for the
```

Comment:
When you run this command your deployment configuration will be updated to match the provided deployment.  By default only the pod and container configuration will be changed and scaling or trigger settings will be left as-is.  Note that environment variables and volumes are included in rollbacks, so if you've recently updated security credentials in your environment your previous deployment may not have the correct values.